dpnp/dpnp_algo/dpnp_algo.pyx

```diff
@@ -227,24 +227,26 @@ cpdef dpnp_queue_is_cpu():
 Internal functions
 """
 cdef DPNPFuncType dpnp_dtype_to_DPNPFuncType(dtype):
+    dt = numpy.dtype(dtype).char
 
-    if dtype in [numpy.float64, numpy.float, 'float64', 'float', 'f8']:
+    if dt == 'd':
         return DPNP_FT_DOUBLE
-    elif dtype in [numpy.float32, 'float32', 'f4']:
+    elif dt == 'f':
         return DPNP_FT_FLOAT
-    elif dtype in [numpy.int64, numpy.int, 'int64', 'int', int]:
+    elif dt == 'q':
```

Comment:
You are missing `dt='i'` as well.

Reply:
In NumPy, `'q'` corresponds to `long long`, `'l'` corresponds to `long`, and `'i'` corresponds to `int`.